recipe/0103-poetry-reading-annotations/manifest1.json

```diff
@@ -0,0 +1,101 @@
+{
+   "@context":"http://iiif.io/api/presentation/3/context.json",
+   "id":"{{ id.url }}",
+   "type":"Manifest",
+   "label":{
+      "en":[
+         "Anne Sexton, Poetry Reading, 1974 -- Her Kind"
+      ]
+   },
+   "items":[
+      {
+         "id":"{{ id.path }}/canvas/1",
+         "type":"Canvas",
+         "duration":107,
+         "items":[
+            {
+               "id":"{{ id.path }}/canvas/1/paintings",
+               "type":"AnnotationPage",
+               "items":[
+                  {
+                     "id":"{{ id.path }}/canvas/1/painting/1",
+                     "type":"Annotation",
+                     "motivation":"painting",
+                     "body":{
+                        "id":"https://library.harvard.edu/poetry/audio/listeningbooth//PS3537E915A6x1974/Her_Kind.mp3",
+                        "type":"Sound",
+                        "format":"audio/mp3",
+                        "duration":107
+                     },
+                     "target":"{{ id.path }}/canvas/1"
+                  }
+               ],
+               "annotations":[
```

Comment:
I think this still needs to be part of the canvas rather than the AnnotationPage. In practice this means adding:
```
      }
],
```
after line 32 and removing the extra `}` and `]` on line 82 and 83.

Reply:
Now fixed